libs/cardano-data/testlib/Test/Cardano/Data/Arbitrary.hs

```diff
@@ -13,7 +13,7 @@ import Test.Cardano.Ledger.Binary.Arbitrary ()
 import Test.QuickCheck
 
 instance (Arbitrary a, Ord a) => Arbitrary (OSet.OSet a) where
-  arbitrary = fmap (OSet.fromSet . Set.fromList) . shuffle . Set.toList =<< arbitrary
```

Comment:
Curious if this is broken or just inefficient - can't figure out if the result is different with this change :thinking: